corehq/extensions/__init__.py

```diff
@@ -1,4 +1,4 @@
-from corehq.extensions.interface import CommCareExtensions, ExtensionError
+from corehq.extensions.interface import CommCareExtensions, ExtensionError   # noqa
```

Comment:
Typically `noqa` is followed by a specific error or warning code to be ignored. Is there a reason why you're completely turning off flake8 on lines like this in this PR?